### YamlMime:ManagedReference
items:
- uid: System.Workflow.Runtime.WorkflowOwnershipException
  id: WorkflowOwnershipException
  children:
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String,System.Exception)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String,System.Exception)
  - System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  langs:
  - csharp
  name: WorkflowOwnershipException
  nameWithType: WorkflowOwnershipException
  fullName: System.Workflow.Runtime.WorkflowOwnershipException
  type: Class
  summary: "A kivétel fordul elő, amikor a munkafolyamat futásidejű motor kísérli meg egy másik munkafolyamat futásidejű adatbázismotor-példány által jelenleg betöltött munkafolyamat-példányok betöltéséhez. A kivétel történt, amikor a munkafolyamat futásidejű motor kísérli meg a munkafolyamat mentése után a tulajdonjog időkorlátja lejárt a munkafolyamat betöltése közben megadott."
  remarks: "> [!NOTE] >  [!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]A WorkflowOwnershipException adatmegőrzési szolgáltatást, amely támogatja a funkció egyszerre több alkalmazás ugyanaz az adatbázis elérésére használják. Ilyen környezetben a munkafolyamat futtatási példánya lehet, hogy próbál olyan munkafolyamatot, amely már be van töltve a munkafolyamat futásidejű motor egy másik példánya. Például a <xref:System.Workflow.Runtime.Hosting.SQLWorkflowPersistenceService>osztály a WorkflowOwnershipException jelez, amikor ez a helyzet akkor fordul elő.</xref:System.Workflow.Runtime.Hosting.SQLWorkflowPersistenceService>       Ha megvalósítása, amely a WorkflowOwnershipException jelez tárolószolgáltatása meg kell adni a <xref:System.Guid>a munkafolyamat-példány, amelynek a kivétel egy megfelelő konstruktor a osztályhoz vagy a beállítás használatával történik a <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>tulajdonság előtt, a kivétel throw.</xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Guid>"
  syntax:
    content: >-
      [System.Obsolete("The System.Workflow.* types are deprecated.  Instead, please use the new types from System.Activities.*")]

      public class WorkflowOwnershipException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor
  id: '#ctor'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException()
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException()
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException()
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> osztály."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonság az új példány egy rendszer által megadott üzenetet, amely leírja a hibát, és figyelembe veszi a rendszer aktuális kulturális környezetének.</xref:System.Exception.Message%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben) |}   | <xref:System.Exception.Message%2A>| A rendszer által megadott honosított leírás. |}   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| An empty <xref:System.Guid>.|</xref:System.Guid></xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A></xref:System.Exception.Message%2A></xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException ();
    parameters: []
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid)
  id: '#ctor(System.Guid)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(Guid)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> osztály <xref:System.Guid>.</xref:System.Guid> megadott munkafolyamat-példány használatával"
  remarks: "`instanceId`megfelel a <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>tulajdonság a munkafolyamat-példány.</xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName> A <xref:System.Exception.Message%2A>az új példány tulajdonság az inicializáláskor felvesz a rendszer által megadott üzenetet, amely leírja a hibát, és figyelembe veszi a rendszer aktuális kulturális környezetének.</xref:System.Exception.Message%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben) |}   | <xref:System.Exception.Message%2A>| A rendszer által megadott honosított leírás. |}   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| A <xref:System.Guid>a munkafolyamat-példány, amelyet `instanceId`, ez a kivétel történt a. |}</xref:System.Guid> </xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (Guid instanceId);
    parameters:
    - id: instanceId
      type: System.Guid
      description: "A <xref:System.Guid>a munkafolyamat-példány, amelyre ez a kivétel történt.</xref:System.Guid>"
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> osztály egy adott hibaüzenet használatával."
  remarks: "A tartalmát a `message` paraméter az célja, hogy ember által is értelmezhető. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított. Ha ezt a konstruktort explicit módon célszerű <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>ahhoz, hogy throw a kivétel.</xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben). |}   | <xref:System.Exception.Message%2A>| A üzenetet hiba karakterlánca. |}   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| An empty <xref:System.Guid>.|</xref:System.Guid></xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A></xref:System.Exception.Message%2A></xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Az üzenet, amely leírja a hibát."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String)
  id: '#ctor(System.Guid,System.String)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(Guid,String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> megadott munkafolyamat-példány használatával osztály <xref:System.Guid>és a megadott hibaüzenet jelenik meg.</xref:System.Guid>"
  remarks: "`instanceId`megfelel a <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>tulajdonság a munkafolyamat-példány.</xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName> A tartalmát a `message` paraméter az célja, hogy ember által is értelmezhető. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben) |}   | <xref:System.Exception.Message%2A>| A üzenetet hiba karakterlánca. |}   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| A <xref:System.Guid>a munkafolyamat-példány, amelyet `instanceId`, ez a kivétel történt a. |}</xref:System.Guid> </xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (Guid instanceId, string message);
    parameters:
    - id: instanceId
      type: System.Guid
      description: "A <xref:System.Guid>a munkafolyamat-példány, amelyre ez a kivétel történt.</xref:System.Guid>"
    - id: message
      type: System.String
      description: "Az üzenet, amely leírja a hibát."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "A szerializált adatok WorkflowOwnershipException osztály egy új példányát inicializálja."
  remarks: "Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum szerializálása során.       További információ: XML- és a SOAP-szerializálási."
  syntax:
    content: protected WorkflowOwnershipException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> , amely tartalmazza a kivétel lépett fel az éppen szerializált objektum adatait."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>, amely a cél- és környezeti információkat tartalmaz.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> osztály megadott hibaüzenet és a belső kivételben találhatók okozza-e ezt a kivételt, mutató hivatkozás segítségével."
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság, amely a konstruktorban, vagy null (Nothing a Visual Basic) hivatkozás van megadva, ha ugyanazt az értéket adja vissza a `innerException` paraméter nem adja meg a belső kivétel értékét a konstruktor.</xref:System.Exception.InnerException%2A> A tartalmát a `message` paraméter az célja, hogy ember által is értelmezhető. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított. Ha ezt a konstruktort explicit módon célszerű <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>ahhoz, hogy throw a kivétel.</xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A belső kivétel referencia. |}   | <xref:System.Exception.Message%2A>| A üzenetet hiba karakterlánca. |}   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| An empty <xref:System.Guid>.|</xref:System.Guid></xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A></xref:System.Exception.Message%2A></xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Az üzenet, amely leírja a hibát."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem egy null hivatkozás (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> a Visual Basic nyelvben), a jelenlegi kivétel következik be egy catch blokkban, amely kezeli a belső kivételben találhatók."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String,System.Exception)
  id: '#ctor(System.Guid,System.String,System.Exception)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(Guid,String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> megadott munkafolyamat-példány használatával osztály <xref:System.Guid>, a megadott hibaüzenet jelenik meg, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt,.</xref:System.Guid>"
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság, amely a konstruktorban, vagy null (Nothing a Visual Basic) hivatkozás van megadva, ha ugyanazt az értéket adja vissza a `innerException` paraméter nem adja meg a belső kivétel értékét a konstruktor.</xref:System.Exception.InnerException%2A>       `instanceId`megfelel a <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>tulajdonság a munkafolyamat-példány.</xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName> A tartalmát a `message` paraméter az célja, hogy ember által is értelmezhető. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A belső kivétel referencia. |}   | <xref:System.Exception.Message%2A>| A üzenetet hiba karakterlánca. |}   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| A <xref:System.Guid>a munkafolyamat-példány, amelyet `instanceId`, ez a kivétel történt a. |}</xref:System.Guid> </xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (Guid instanceId, string message, Exception innerException);
    parameters:
    - id: instanceId
      type: System.Guid
      description: "A <xref:System.Guid>a munkafolyamat-példány, amelyre ez a kivétel történt.</xref:System.Guid>"
    - id: message
      type: System.String
      description: "Az üzenet, amely leírja a hibát."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem egy null hivatkozás (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> a Visual Basic nyelvben), a jelenlegi kivétel következik be egy catch blokkban, amely kezeli a belső kivételben találhatók."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
  type: Method
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Beállítja a <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> rendelkező objektum a <xref:System.Guid>ezt a kivételt, és további információ a kivételről társított munkafolyamat-példány.</xref:System.Guid>"
  remarks: "GetObjectData eljárás beállítja a <xref:System.Runtime.Serialization.SerializationInfo>összes kivétel objektum adatokkal szerializálási szánt.</xref:System.Runtime.Serialization.SerializationInfo> A deszerializálás során a kivétel objektum előállított a <xref:System.Runtime.Serialization.SerializationInfo>az adatfolyam továbbított.</xref:System.Runtime.Serialization.SerializationInfo>       További információ: XML- és a SOAP-szerializálási."
  syntax:
    content: public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> , amely tartalmazza a kivétel lépett fel az éppen szerializált objektum adatait."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>, amely a cél- és környezeti információkat tartalmaz.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "<code>info</code>null hivatkozás (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> a Visual Basic nyelvben)"
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  id: InstanceId
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: InstanceId
  nameWithType: WorkflowOwnershipException.InstanceId
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  type: Property
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Lekérdezi a <xref:System.Guid>a munkafolyamat-példány, amelyre kivétel történt.</xref:System.Guid>"
  syntax:
    content: public Guid InstanceId { get; set; }
    return:
      type: System.Guid
      description: "InstanceId értéke megegyezik a &lt;xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName&gt; tulajdonság a munkafolyamat-példány."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException()
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException()
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException()
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(Guid)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid)
- uid: System.Guid
  parent: System
  isExternal: true
  name: Guid
  nameWithType: Guid
  fullName: System.Guid
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(Guid,String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String,System.Exception)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String,System.Exception)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(Guid,String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: InstanceId
  nameWithType: WorkflowOwnershipException.InstanceId
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException
- uid: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData*
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: GetObjectData
  nameWithType: WorkflowOwnershipException.GetObjectData
- uid: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId*
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: InstanceId
  nameWithType: WorkflowOwnershipException.InstanceId
